src/utils.js

```diff
@@ -1,82 +1,112 @@
-}
-
-function showHelp() {
-  console.log(USAGE);
-}
+async function cloneGitRepo(commandName, projectName, options) {
+  // const clipDest = path.join("./", projectName, "clip");
+  // const clipFrom = path.join("./", projectName, "dist", "clip", "**", "*");
+  // await copyfiles([clipFrom, clipDest], { up: 3 });
 
-function cloneGitRepo(command, projectName) {
-  console.log(MESSAGES.clone);
+  // return;
+  let throbber = ora("Initializing...").start();
   try {
+    const execOpts = {
+      cwd: projectName,
+    };
+    //check if folder name exists
     if (fs.existsSync(projectName)) {
-      console.log(MESSAGES.folderExists(projectName));
+      throbber.stop();
+      console.error(MESSAGES.folderExists(projectName));
       return;
     }
-  } catch (e) {
-    console.log(MESSAGES.unexpectedError, e);
-  }
-  const dir = path.join(process.cwd(), projectName);
-  Git.clone({
-    fs,
-    http,
-    dir,
-    url: STARTER_REPO[command],
-  })
-    .then(function () {
-      // remove git files
-      if (process.platform === "win32") {
-        rimraf(`.\\${projectName}\\.git`, () => console.log("Almost ready.."));
-        rimraf(`.\\${projectName}\\.github`, () =>
-          console.log("Cleaning things up..")
-        );
-      } else {
-        rimraf(`./${projectName}/.git`, () => console.log("Almost ready.."));
-        rimraf(`./${projectName}/.github`, () =>
-          console.log("Cleaning things up..")
-        );
-      }
-      const throbber = ora(MESSAGES.install).start();
-      exec(`npm install`, { cwd: projectName }, (error, stdout, stderr) => {
-        if (error) {
-          throbber.stop();
-          console.error(error.message);
-          return;
-        }
-        if (stderr) {
-          console.error(stderr);
-        }
+    throbber.stop();
+
+    //CLONE THE REPO
+    throbber = ora(MESSAGES.clone).start();
+    const dir = path.join(process.cwd(), projectName);
+    await Git.clone({
+      fs,
+      http,
+      dir,
+      url: STARTER_REPO[commandName],
+    });
+    throbber.stop();
+
+    //CLEAN GITHUB FOLDERS
+    throbber = ora("Cleaning things up..").start();
+    const gitFile = path.join("./", projectName, ".git");
+    const githubFile = path.join("./", projectName, ".github");
 
+    await rimraf(gitFile);
+    await rimraf(githubFile);
+    throbber.stop();
+
+    //INSTALL DEPENDECIES
+    throbber = ora(MESSAGES.install).start();
+    let command = "npm i";
+    if (!options.tsc) command = `npm i && npx tsc`;
+    await exec(command, execOpts);
+    throbber.stop();
+
+    //ADD JS FILES IF NOT TS
+    if (options.js) {
+      throbber = ora(MESSAGES.javascript).start();
+
+      const clipDest = path.join("./", projectName, "clip");
+      const clipFrom = path.join("./", projectName, "dist", "clip", "**", "*");
+      await copyfiles([clipFrom, clipDest], { up: 3 });
+
+      const serverDest = path.join("./", projectName, "server");
+      const serverFrom = path.join(
+        "./",
```

Comment:
Shouldn't this be the same?
```suggestion
        ".",
```